nuxeo-java-client-test/src/test/java/org/nuxeo/client/test/TestCurrentUser.java

```diff
@@ -26,21 +26,28 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.nuxeo.client.api.objects.user.CurrentUser;
+import org.nuxeo.client.api.objects.user.User;
+import org.nuxeo.client.api.objects.user.UserManager;
 import org.nuxeo.client.internals.spi.NuxeoClientException;
+import org.nuxeo.client.internals.spi.auth.PortalSSOAuthInterceptor;
 import org.nuxeo.ecm.core.test.annotations.Granularity;
 import org.nuxeo.ecm.core.test.annotations.RepositoryConfig;
 import org.nuxeo.ecm.restapi.test.RestServerFeature;
-import org.nuxeo.runtime.test.runner.Features;
-import org.nuxeo.runtime.test.runner.FeaturesRunner;
-import org.nuxeo.runtime.test.runner.Jetty;
+import org.nuxeo.ecm.restapi.test.RestServerInit;
+import org.nuxeo.runtime.test.runner.*;
+
+import java.util.ArrayList;
+import java.util.List;
 
 /**
  * @since 0.1
  */
 @RunWith(FeaturesRunner.class)
 @Features({ RestServerFeature.class })
 @Jetty(port = 18090)
-@RepositoryConfig(cleanup = Granularity.METHOD)
+@Deploy({ "org.nuxeo.ecm.webengine.core", "org.nuxeo.ecm.automation.server", "org.nuxeo.ecm.platform.login.portal" })
```

Comment:
`org.nuxeo.ecm.automation.server` isn't needed because it is deployed by the `RestServerFeature` via the `EmbeddedAutomationServerFeature`

`org.nuxeo.ecm.webengine.core` isn't needed for the same reason, via the `WebEngineFeature`